loopbackStarter/README.md

```diff
@@ -12,21 +10,38 @@ $ npm install -g strongloop
 If you run into any issues, see [Installing StrongLoop](http://docs.strongloop.com/display/LB/Installing+StrongLoop)
 for more information.
 
+You must have already installed the [Cloud Foundry Command Line Interface (cf CLI)](https://docs.cloudfoundry.org/cf-cli/) for deploying the app to Bluemix.
+
+## Deploying to Bluemix
+
+**Prerequisites**
+
+This app is pre-configured to be deployed to Bluemix, however, the following prerequisites must be met before it can be run successfully.
+
+- Modify `manifest.yml` and `package.json` to replace "loopback-starter-app" with a unique name for your app instance.
+- Push the app to Bluemix using `cf push <name of your app> --no-start`.
+- Create a [MongoDB](https://console.ng.bluemix.net/catalog/services/compose-for-mongodb) named `loopback-mongodb-starter`, and bind the service to your app instance.
+- If you haven't bound `loopback-mongodb-starter` to your app while provisioning it, you can bind the service to the app using the `cf bind-service` command.
+
```

Comment:
All of these instructions can be simplified to just:
### Deployment
- Modify `manifest.yml` and `package.json` to replace "loopback-starter-app" with a unique name for your app instance.
- Create a mongodb service: 
```
cf create-service compose-for-mongodb Standard loopback-mongodb-starter
```
- Push the app to Bluemix using `cf push <name of your app>`

-------------------
We can skip the step to bind the service by adding a line to manifest.yml (see comment)